terraform/modules/fourkeys/README.md

```diff
@@ -23,18 +23,43 @@ No providers.
 
 | Name | Source | Version |
 |------|--------|---------|
-| <a name="module_bigquery"></a> [bigquery](#module\_bigquery) | ../fourkeys-bigquery | n/a |
+| <a name="module_circleci_parser"></a> [circleci\_parser](#module\_circleci\_parser) | ../fourkeys-circleci-parser | n/a |
 | <a name="module_cloud_build_parser"></a> [cloud\_build\_parser](#module\_cloud\_build\_parser) | ../fourkeys-cloud-build-parser | n/a |
-| <a name="module_dashboard"></a> [dashboard](#module\_dashboard) | ../fourkeys-dashboard | n/a |
-| <a name="module_foundation"></a> [foundation](#module\_foundation) | ../fourkeys-foundation | n/a |
 | <a name="module_fourkeys_images"></a> [fourkeys\_images](#module\_fourkeys\_images) | ../fourkeys-images | n/a |
 | <a name="module_github_parser"></a> [github\_parser](#module\_github\_parser) | ../fourkeys-github-parser | n/a |
 | <a name="module_gitlab_parser"></a> [gitlab\_parser](#module\_gitlab\_parser) | ../fourkeys-gitlab-parser | n/a |
+| <a name="module_pagerduty_parser"></a> [pagerduty\_parser](#module\_pagerduty\_parser) | ../fourkeys-pagerduty-parser | n/a |
 | <a name="module_tekton_parser"></a> [tekton\_parser](#module\_tekton\_parser) | ../fourkeys-tekton-parser | n/a |
 
 ## Resources
 
-No resources.
+| Name | Type |
+|------|------|
+| [google_bigquery_dataset.four_keys](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_dataset) | resource |
```

Comment:
Not sure where the bigquery resources are coming from

Reply:
this is auto-generated by `terraform-docs` and based on the resource generated by the module. This dataset resource is here: https://github.com/GoogleCloudPlatform/fourkeys/blob/main/terraform/modules/fourkeys/bigquery.tf#L8-L15

Note: This isn't a change in this PR, just `terraform-docs` hadn't been run since the module was updated.